index.js

```diff
@@ -16,4 +16,268 @@ function closeSideMenu() {
   logoName.style.display = 'block';
 }
 
-closeSideMenu();
\ No newline at end of file
+closeSideMenu();
+
+const windowSize = window.matchMedia('(min-width: 1024px)');
+
+const project = [
```

Comment:
- [ ] [OPTIONAL] Kindly move this project array in another  file , eg: data.js and import it into your index.js so as to keep your code in your index.js more organized